test_builder/test_builder

```diff
@@ -698,7 +698,7 @@ function set_parameter_default() {
 function grpc_protocol_driver_setup() {
   set_parameter_default client_type "polling" "callback polling"
   set_parameter_default server_type "inline" "handoff inline polling"
-  set_parameter_default threadpool_size "4" "0, 1, 2, 3, etc. and -1 for NumCPU() threads."
+  set_parameter_default threadpool_size "-1" "0, 1, 2, 3, etc. and -1 for NumCPU() threads."
```

Comment:
do we allow zero for threadpool_size? what does that mean? Perhaps zero should mean NumCpus() instead of -1

Reply:
Done